res2df/summary.py

```diff
@@ -801,7 +804,7 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     """
     parser.add_argument(
         "DATAFILE",
-        help="Name of Eclipse DATA file. " + "UNSMRY file must lie alongside.",
+        help="Name of reservoir DATA file. " + "UNSMRY file must lie alongside.",
```

Comment:
Can remove `" + "` as it is all on the same line. Also, as previously mentioned, I think we should maybe use `Simulator input .DATA file` instead of `reservoir DATA file`.